service/src/main/scala/za/co/absa/loginsvc/rest/config/auth/UsersConfig.scala

```diff
@@ -56,22 +56,16 @@ case class UsersConfig(knownUsers: Array[UserConfig], order: Int)
     }
     else ConfigValidationSuccess
   }
-
-  @PostConstruct
-  def init(): Unit = {
-    this.validate().throwOnErrors()
-  }
 }
 
-@ConstructorBinding
 case class UserConfig(username: String,
                        password: String,
-                       email: String, // may be null
+                       email: Option[String], // may be null
```

Comment:
```suggestion
                       email: Option[String],
```
Seeing this become an `Option` makes me happy. What about the comment - I assume the value will be `None`, not null when not found.

Reply:
😆 what a fix (yes, an option can be None, but that is sort of expected). The comment is not wrong, but my suggestion was to remove it.

Reply:
Ah, will do